bootstrap.py

```diff
@@ -918,11 +929,49 @@ class MEOptions:
         migrate_systems(options.org, options.activationkey)
         if options.enablerepos:
             enable_repos()
+    elif options.new_capsule and options.foreman_fqdn:
+        # > ELIF new_capsule and foreman_fqdn set, will migrate to other capsule
+        #
+        # > will replace CA certificate, reinstall katello-agent, gofer
+        # > update system definition to point to new capsule for content,
+        # > Puppet, OpenSCAP and update Puppet configuration
+        # > MANUAL SIGNING OF CSR OR MANUALLY CREATING AUTO-SIGN RULE STILL REQUIRED!
+        # > API doesn't have a public provision for creating auto-sign entries yet!
+        get_bootstrap_rpm(clean=True)
+        install_katello_agent()
+        API_PORT = get_api_port()
+
+        print_running("Calling Foreman API to update content source, puppet master, puppet ca and openscap proxy records for %s" % FQDN)
+        capsule_id = return_matching_katello_key('capsules', 'name="%s"' % options.foreman_fqdn, 'id', False)
```

Comment:
This would make (this part of the code) also work on Foreman installs without Katello. Plus I'd hope we can someday get rid of the Katello-specific API completely.

I gave the APIs on 1.15 upstream nightly and 6.2.8 Satellite a shot yesteday, the results looked identical enough for your needs.